poc/vdaf_prio3.sage

```diff
@@ -1,4 +1,11 @@
 # The prio3 VDAF.
```

Comment:
This won't compile because `byte` is not in scope yet. We need to move the constants after the imports.